client.go

```diff
@@ -2119,3 +2166,11 @@ func (fs *FileStat) IsDir() bool {
 func (fs *FileStat) Sys() interface{} {
 	return fs
 }
+
+type FileSecurityInfo struct {
+	Owner *Sid
+	Group *Sid
+	Flags uint16
+	Sacl  []ACE
```

Comment:
same as above. we should define ACE(ACL?) type on the top package instead of internal package.

Reply:
I refactored the way ACE's are decoded. Let me know if you agree with this approach!
An other option is renaming the internal ACE struct and just copy everything to the external ACE.